client/src/App.js

```diff
@@ -13,6 +14,7 @@ function App() {
       <Routes>
         <Route path="/" element={<Shell />} />
```

Comment:
```suggestion
        <Route path="/*" element={<Shell />} />
```
While we're here add this, we missed it earlier.